### PR TITLE
Added note about the Power Apps connector

### PR DESCRIPTION
**Pull request by kaklakariada**

### CODE REVIEW

File: doc/user_guide/user-guide.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+Exasol REST API is an extension for the Exasol database that provides the ability to interact with the database via REST API endpoints. It is required to use the [Power Apps Connector for Exasol](https://github.com/exasol/power-apps-connector).
+
 ## Installation
 
-### Environment
+### Deployment
 
 The REST API needs to be installed on a internet-facing server machine/VM that also has connectivity to the database environment.
 
+The REST API is stateless, so you can install it on multiple nodes put an HTTP load balancer in front.
```
Comment (pj-spoelders) [inline doc/user_guide/user-guide.md:11]:
```suggestion
The REST API is stateless, so you can install it on multiple nodes and put a HTTP load balancer in front.
```

**Review by pj-spoelders — CHANGES_REQUESTED** [review]

**Review by pj-spoelders — APPROVED** [review]

File: doc/user_guide/user-guide.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 # Exasol REST API
 
+Exasol REST API is an extension for the Exasol database that provides the ability to interact with the database via REST API endpoints. It is required to use the [Power Apps Connector for Exasol](https://github.com/exasol/power-apps-connector).
```
Comment (frschwab) [inline doc/user_guide/user-guide.md:3]: better: "E.g. the REST API is required by the [Power Apps Connector for Exasol]."
Then it is clear that it is not the other way around.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Exasol REST API
-[![Build Status](https://github.com/exasol/exasol-rest-api/actions/workflows/ci-build.yml/badge.svg)](https://github.com/exasol/exasol-rest-api/actions/workflows/ci-build.yml)
 
-Exasol REST API is an extension for the Exasol database that provides the ability to interact with the database via REST API endpoints.
+[![Build Status](https://github.com/exasol/exasol-rest-api/actions/workflows/ci-build.yml/badge.svg)](https://github.com/exasol/exasol-rest-api/actions/workflows/ci-build.yml)
 
+Exasol REST API is an extension for the Exasol database that provides the ability to interact with the database via REST API endpoints. It is required to use the [Power Apps Connector for Exasol](https://github.com/exasol/power-apps-connector).
```
[the remark this answers is not in the document]
Comment (kaklakariada) [reply README.md:5]: I implemented your change in https://github.com/exasol/exasol-rest-api/pull/63

File: doc/user_guide/user-guide.md (hunk 2 above)
Comment (kaklakariada) [reply doc/user_guide/user-guide.md:3]: See https://github.com/exasol/exasol-rest-api/pull/63

**Pull request merged**